setup.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+
+RED='\033[0;31m'
+NC='\033[0m'
+python_command="python"
+pip_command="pip"
+
+function checkPythonVersion () {
+    case "$($1 --version 2>&1)" in
+    *" 3."*)
+        return 1
+        ;;
+    *)
+        return 0
+        ;;
+    esac
+}
+
+function checkPIPVersion () {
+    case "$($1 --version 6>&1)" in
+    *" 3."*)
+        return 1
+        ;;
+    *)
+        return 0
+        ;;
+    esac
+}
+
+function main () {
+    if checkPythonVersion $python_command; then
+        python_command="python3"
+        if checkPythonVersion $python_command; then
+            echo "Python3 not found."
+            exit 1
+        fi
+        echo "Using python3 instead of python"
+    fi
+
+    if checkPIPVersion $pip_command; then
+        pip_command="pip3"
+        if checkPIPVersion $pip_command; then
+            echo "pip for Python3 not found."
+            exit 1
+        fi
+        echo "Using pip3 instead of pip"
+    fi
+
+    $pip_command install -r requirements.txt
+}
+
+function full () {
+    echo -e "Running the ${RED}full${NC} setup (including building the model)"
+    main
+    mkdir data
+    curl https://transfer.sh/V7mkg/recipe_raw.tsv -o data/recipe_raw.tsv
```

Comment:
we can use a global variable for the download link since it might change again due to timeout/expiry for `transfer.sh`, Since `transfer.sh` is no longer developed, we will probably change to other methods. [transfer.sh alternative](https://github.com/dutchcoders/transfer.sh/issues/116)